examples/third-parties/codesandbox/package.json

```diff
@@ -5,6 +5,7 @@
   "engines": {
     "node": ">= 12"
```

Comment:
Please change to Node.js 16+ because we are switching to ESM, and 14 is the minimum requirement. (16 is for the future)

Reply:
Okay, I'll do it